deepchecks/base/display_suite.py

```diff
@@ -55,13 +54,13 @@ def get_display_exists_icon(exists: bool):
 
 def display_suite_result(suite_name: str, results: List[Union[CheckResult, CheckFailure]]):
     """Display results of suite in IPython."""
-    conditions_table = []
+    checks_with_conditions = []
     display_table = []
     others_table = []
     for result in results:
         if isinstance(result, CheckResult):
             if result.have_conditions():
-                ConditionResult.append_to_conditions_table(result, conditions_table)
```

Comment:
If you sort at the end we can't keep the conditions ordered by the checks we run

Reply:
I don't understand the problem can you elaborate?

Reply:
oh I mistook it with another function